app/state/reducers/searchFiltersReducer.js

```diff
@@ -0,0 +1,18 @@
+const initialState = {
+  search: '',
+};
+
+function searchFiltersReducer(state = initialState, action) {
+  switch (action.type) {
+    case 'ENTER_OR_CHANGE_SEARCH_PAGE': {
```

Comment:
But if we start using { } around cases, we need an eslint rule for that. Otherwise I think we'll just forget in a lot of cases.